scripts/merge-PR.sh

```diff
@@ -16,6 +16,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# Setting the script to fail if anything goes wrong
+set -e
 
 # Use this to simplify the rebasing of PRs. PRs will be rebased during the merge on this process.
 # use: ./rebase-PR <PRID> textual description
```

Comment:
@johnament I wrote this to simplify how I was doing prs.. so others could do the same.

I'm curious now what is this rebase-PR that you mentioned.